RInAppMessaging/Classes/Repositories/CampaignRepository.swift

```diff
@@ -151,8 +159,6 @@ internal class CampaignRepository: CampaignRepositoryType {
     private func saveDataToCache(_ list: [Campaign]) {
         let user = preferenceRepository.getUserIdentifiers()
         userDataCache.cacheCampaignData(list, userIdentifiers: user)
-        if defaultCacheUser != user {
-            userDataCache.cacheCampaignData(list, userIdentifiers: defaultCacheUser)
-        }
+        userDataCache.cacheCampaignData(list, userIdentifiers: CampaignRepository.lastUser)
```

Comment:
Why is the same campaign list cached twice?

Reply:
It's basically the same approach as before but it doesn't store to anonymous user cache (to avoid userA -> logout -> userB data transfer)